tests/dummy/app/routes/users/detail.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
```

Comment:
Since the parent route uses the AuthenitcatedRouteMixin, this one probably should too.